packages/arb-ts/src/lib/networks.ts

```diff
@@ -71,6 +73,7 @@ const networks: Networks = {
     isArbitrum: true,
     tokenBridge: mainnetBridge,
     confirmPeriodBlocks: 45818,
+    rpcURL: 'https://arb1.arbitrum.io/rpc',
```

Comment:
Should we allow an option for the user to override it?
`rpcURL: process.env['L2RPC'] || 'https://arb1.arbitrum.io/rpc'`